packages/puppeteer-core/src/api/Browser.ts

```diff
@@ -461,6 +461,19 @@ export abstract class Browser extends EventEmitter<BrowserEvents> {
     return await this.defaultBrowserContext().deleteCookie(...cookies);
   }
 
+  /**
+   * Installs an extension and returns the ID. Available if the browser was
```

Comment:
This API would work for Firefox and other browsers potentially, let's make sure that these specific requirements mention Chrome specifically.

Reply:
In fact, Firefox WebDriver BiDi does not work with pipes so it could be interpreted wrong.

Reply:
I went ahead and updated the comment :)